logger/index.js

```diff
@@ -173,19 +173,26 @@ transmitter.on('glucose', glucose => {
     });
 });
 
+transmitter.on('sawTransmitter', data => {
+  const util = require('util')
+  console.log('got sawTransmitter message inside logger msg: ' + JSON.stringify(data));
+  console.log(util.inspect(data, false, null))
+
+  var fs = require('fs');
+  const battery = JSON.stringify(data);
```

Comment:
Is `battery` the left over name from copy/paste?

Reply:
Yes, good catch. I'll fix that.